### PR TITLE
Add mart-e.be to 2024

### PR DESCRIPTION
**Pull request by mart-e**
For posterity https://web.archive.org/web/20240409064540/https://mart-e.be/

### CODE REVIEW

File: 2024.html [hunk 1]
```diff
@@ -70,6 +70,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://emmajuettner.com/">Emma Juettner</a></li>
             <li><a href="https://eklausmeier.goip.de/blog">Elmar Klausmeier</a></li>
             <li><a href="https://knowler.dev/">Nathan Knowler</a></li>
+            <li><a href="https://mart-e.be/">Martin Trigaux</a></li>
```
Comment (arkhi) [inline 2024.html:73]: Hello! Can you sort by family name instead (_Trigaux_ would come after _Proto_)? Thanks!

Comment (mart-e) [reply 2024.html]: oh ok, I thought somehow it was sorted by url. Actually changed it to the name of the blog as my full name doesn’t even appear on the website so found it a bit odd

**Review by arkhi — APPROVED** [review]
Thanks! Although I believe `mart-e` would come before `Massart`. :)

I made the change and will merge.

Welcome in!

**Comment by mart-e** [comment]
Not enough coffee, thanks !

**Pull request merged**